content-review/impl/turnitin-oc/src/main/java/org/sakaiproject/contentreview/turnitin/oc/ContentReviewServiceTurnitinOC.java

```diff
@@ -260,18 +253,13 @@ public String setupWebhook(String webhookUrl) throws Exception {
 
 		if ((responseCode < 200) || (responseCode >= 300) || (responseBody == null)) {
 			throw new Error(responseMessage);
-		} else {
-			
-			if (responseBody != "" && responseBody != "[]") {
-				// Make sure the webhook(s) is/are valid
-				// Create List from responseBody
-				
-				JSONArray webhookList = JSONArray.fromObject(responseBody);
-				for (int i=0; i < webhookList.size(); i++) {
-					JSONObject webhookJSON = webhookList.getJSONObject(i);
-					if (webhookJSON.has("id") && webhookJSON.has("url")) {
-						webhooks.add(new Webhook(webhookJSON.getString("id"), webhookJSON.getString("url")));
-					}
+		} else if (responseBody != "" && responseBody != "[]") {
```

Comment:
Don't use "=" when comparing strings in Java
else if(StringUtils.isNotEmpty(responseBody) && !"[]".equals(responseBody))